apps/tup-ui/src/pages/Dashboard/Dashboard.module.css

```diff
@@ -29,7 +29,7 @@
     grid-template-columns: 0.5fr 0.5fr;
   }
   @media (--x-wide-and-above) {
-    grid-template-columns: 0.65fr 0.35fr;
+    grid-template-columns: 0.6fr 0.4fr;
```

Comment:
This performs #369 **again**, but is tested at 100% zoom. ✔️ #369 was tested at 90% zoom. 🫤